services/website/website.service.js

```diff
@@ -95,9 +95,17 @@ module.exports = class Website extends LegacyService {
   static get examples() {
     return [
       {
-        previewUrl: 'website-up-down-green-red/https/shields.io',
-        queryParams: { label: 'my-website' },
-        keywords: ['website'],
+        title: 'Website',
+        pattern: 'website-up-down-green-red/:protocol/:hostAndPath',
```

Comment:
Also the `up-down-green-red` part is optional on this badge right? I'm not sure what the pattern would need to look like to make that part configurable in the modal (maybe `website-:(options)/:protocol/...` with `up-down-green-red` as the value for the named `options` param?), but is there value in allowing the user to enter their desired options in the model to build the badge urls?

Reply:
Yea… I tried messing with the pattern for a little while and it seems like supporting this format is going to require further changes to the frontend. I'd prefer to limit the kinds of patterns we use, and move options like that to the query string.

e.g. `?up_message=&down_message=&up_color=&down_color=`

Besides reducing the amount of custom code needed to build the URLs, it has an added benefit of keeping our badge APIs relatively uniform.

Reply:
I like the sound of that